src/Libraries/Microsoft.Extensions.AI.Abstractions/SpeechToText/SpeechToTextOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>Represents the options for an speech to text request.</summary>
+[Experimental("MEAI001")]
+public class SpeechToTextOptions
+{
+    /// <summary>Gets or sets the ID for the speech to text.</summary>
+    /// <remarks>Long running jobs may use this ID for status pooling.</remarks>
```

Comment:
status pooling => status polling

How does someone use this for polling?

Does this / how does this related to ChatOptions.ChatThreadId?

Is the idea you can somehow specify a previously received response ID and continue it somehow?

Reply:
Removed the property, this was added to address a problem that doesn't exists for 99% of use cases.